Plugin/Model/ReplacePasswordValidationRules.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+namespace Aligent\Pci4Compatibility\Plugin\Model;
+
+use Magento\Framework\Validator\DataObject;
+use Magento\Framework\Validator\NotEmpty;
+use Magento\Framework\Validator\Regex;
+use Magento\Framework\Validator\StringLength;
+use Magento\User\Model\UserValidationRules;
+
+class ReplacePasswordValidationRules
+{
+
+    private const MIN_PASSWORD_LENGTH = 12;
+
+    /**
+     * Updates the requirements for admin passwords so that the minimum length is 12 characters
+     *
+     * @param UserValidationRules $subject
+     * @param callable $proceed
+     * @param DataObject $validator
+     * @return DataObject
+     */
+    public function aroundAddPasswordRules(
```

Comment:
If you review the PR below, there are a couple of changes to the test scenarios. Do these changes impact the tests?

https://github.com/magento/magento2/pull/39319/files

Reply:
Any change going in there would not be included in 2.4.6 or 2.4.7 (other than potentially via a quality patch.
I can see the validation.js script has been updated, so I can take a look at fixing that up too, though JS changes are always messier.
Changes to Magento's own tests are irrelevant, as we are only going to be running our own test cases.